### PR TITLE
[eclipse/xtext#1954] turn of pack200 by default

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext#1954] turn of pack200 by default
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: releng/jenkins/sign-and-deploy/Jenkinsfile [hunk 1]
```diff
@@ -96,7 +96,7 @@ spec:
       ''')
     string      (name: 'BRANCH_TO_DEPLOY', defaultValue: 'master', description: 'From which Git branch should the release be created (master for snapshot deployments)?')
     booleanParam(name: 'ORG_GRADLE_PROJECT_OSSPUB_SIGN_JARS', defaultValue: true, description: 'Whether to sign jars using the Eclipse web service')
-    booleanParam(name: 'ORG_GRADLE_PROJECT_OSSPUB_PACK_JARS', defaultValue: true, description: 'Whether to pack jars using pack200')
+    booleanParam(name: 'ORG_GRADLE_PROJECT_OSSPUB_PACK_JARS', defaultValue: false, description: 'Whether to pack jars using pack200')
```
Comment (cdietrich) [inline releng/jenkins/sign-and-deploy/Jenkinsfile:99]: maybe we should also completely hardcode this.
i dont know if there is any cbi verifier whatever that still wants packed files

**Comment by cdietrich** [comment]
i assume an alternative fix would be to teach the publishing to create two entries with and without pack

**Comment by cdietrich** [comment]
TODO: do we already want to try this with the 2.25 release

**Comment by szarnekow** [comment]
> TODO: do we already want to try this with the 2.25 release

Yes, I think so.

**Review by szarnekow — APPROVED** [review]

**Comment by cdietrich** [comment]
@merks 

your report
https://download.eclipse.org/oomph/archive/reports/download.eclipse.org/staging/2021-03/index.html

reports missing pack.gz files.
how much of a requirement is it to have them?

Bildschirmfoto 2021-02-23 um 11 55 40

**Comment by merks** [comment]
Here is a SimRel requirement to ship such things, but it's not really enforced and given that as of Java 15 it's no longer a usable artifact, it doesn't make sense to even have the rule in the long term.  This won't lead to an error marker on your overall simrel contribution; those only come from unsigned content or bad license.

**Review by NicoPrediger — APPROVED** [review]

**Pull request merged**